components/brave_wallet_ui/page/container.tsx

```diff
@@ -119,6 +130,63 @@ function Container (props: Props) {
 
   const recoveryPhrase = (mnemonic || '').split(' ')
 
+  // Will need to add additional logic here and in the reducer
+  // to fetch price history once the pricing api is read.
+  const onChangeTimeline = (timeline: ChartTimelineType) => {
+    props.walletPageActions.changeTimline(timeline)
+  }
+
+  // This will scrape all of the user's accounts and combine the asset balances for a single asset
+  const fullAssetBalance = (asset: AssetOptionType) => {
```

Comment:
nit: prefer a verb for these variables to make it clear they're not storing anything but a function, e.g. `calculateFullAssetBalance` and `calculateFullFiatBalance`. Also consider moving these to reducer, if you want all your state balance / accounts manipulation in one place, but this is fine too as long as memoized like you've done nicely 👍